diff_drive_controller/include/diff_drive_controller/velocity_limiter.hpp

```diff
@@ -0,0 +1,47 @@
+/*
+ * velocity_limiter.hpp
+ *
+ *  Created on: May 30, 2019
+ *      Author: Christof Dubs
+ */
+
+#pragma once
+
+namespace diff_drive_controller {
+
+class VelocityLimiter {
+public:
+    struct Vector {
+        float x;
+        float th;
+    };
+
+    VelocityLimiter();
+
+    bool init(float wheel_separation, bool has_velocity_limits = false, bool has_acceleration_limits = false,
+            float vel_x_max = 0.0, float _vel_th_max = 0.0, float acc_x_max = 0.0, float acc_x_min = 0.0,
```

Comment:
doesn't really matter but "_vel_th_max", also consider defining a struct for the set of param

Reply:
will do.